standard/expressions.md

```diff
@@ -1275,6 +1278,36 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 > ```
 > Within the `A` class, those occurrences of the Color identifier that reference the Color type are delimited by `**`, and those that reference the Color field are not. *end example*
 
+### §null-conditional-member-access Null Conditional Member Access
+
+A *null_conditional_member_access* consists of a *primary_expression* followed by a "`?.`" token, followed by an *Identifier*, optionally followed by a *type_argument_list*.
```

Comment:
I don't believe `?.` is one token.  See https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDZICYgNQA+AAgEwCMAsAFBEDMABKfQML0De19XjDRALPQCyACiJkADPSgBKdp24KAbhAT0M9ALxSA/PQB0AGTgA7AOYwAFgG55XAL7U7QA

Reply:
Hmmm... Is there a good reason to allow spaces or is this an artefact of the way the compiler added the token? The token `??` already exists (as do other multi-symbol tokens where a prefix is also a token), does the compiler allow a space? Historically there was the change to break up right shift (assignment) due to the conflict when generics were introduced, but there is no such conflict between `?`, `??`, `?.` and `?[` is there?

If it is technically a breaking change to make it a single token I really hope it doesn't effect anyone (we might never know of course as they could be too embarrassed to admit it!)

I'd encourage we go for the single token, but its not a hill I'll die on! What do folks think? (The question applies to both PR #7 and this one).

Reply:
Changed